src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
```

Comment:
Similarly to the previous comment, replace the literal `20` with a named constant to explain what this value represents. For instance, `SHORT_TERM_DISCOUNT` could be used here. Refer to checklist item 2.